### PR TITLE
CPF-277 #comment changed to validate input fields from user model

### PR DESCRIPTION
**Pull request by noodlemoodle**

### CODE REVIEW

**Comment by Nauna** [comment]
tested with corresponding PR: https://github.com/nanoPayinc/NANOPAY/pull/6692

and all works wonderfully 🥇

File: src/foam/nanos/auth/User.js [hunk 1]
```diff
@@ -95,15 +95,33 @@ foam.CLASS({
...
       class: 'String',
       name: 'middleName'
     },
     {
       class: 'String',
-      name: 'lastName'
+      name: 'lastName',
+      validateObj: function(lastNameField) {
+        if (!(lastNameField.trim())){
+          return 'Last Name Field Required.'
```
Comment (jlhughes) [inline src/foam/nanos/auth/User.js]:
```suggestion
          return 'Last Name Required.'
```

File: src/foam/nanos/auth/User.js [hunk 2]
```diff
@@ -114,8 +132,10 @@ foam.CLASS({
       width: 100,
       tableWidth: 160,
       validateObj: function(organization) {
-        if ( organization.length > 35 ) {
-          return 'Organization name cannot exceed 35 characters.';
+        if ( organization.length > 70 ) {
+          return 'Company name cannot exceed 70 characters.';
+        } if (!(organization.trim())) {
+          return 'Company Name Field Required.';
```
Comment (jlhughes) [inline src/foam/nanos/auth/User.js]:
```suggestion
          return 'Company Name Required.';
```

File: src/foam/nanos/auth/User.js [hunk 3]
```diff
@@ -140,10 +160,13 @@ foam.CLASS({
        emailIsSet_ = true;`,
       validateObj: function (email) {
         var emailRegex = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-
-        if ( ! emailRegex.test(email) ) {
-          return 'Invalid email address.';
+        
+        if (!(email.trim())) {
+          return 'Email Field Required.';
```
Comment (jlhughes) [inline src/foam/nanos/auth/User.js]:
```suggestion
          return 'Email Required.';
```

**Review by jlhughes — CHANGES_REQUESTED** [review]

**Pull request merged**